website/docs/r/database_user.html.markdown

```diff
@@ -98,6 +98,11 @@ Accepted values include:
   * `USER` - New database user has AWS IAM user credentials.
   * `ROLE` -  New database user has credentials associated with an AWS IAM role.
 
+* `ldap_auth_type` - (Optional) Method by which the provided username is authenticated. If no value is given, Atlas uses the default value of NONE.
+  * `NONE` -	Atlas authenticates this user through SCRAM-SHA, not LDAP.
+  * `USER` - LDAP server authenticates this user through the user's LDAP user. `username` must also be a fully qualified distinguished name, as defined in RFC-2253.
```

Comment:
Please include the RFC link from the source docs - https://tools.ietf.org/html/2253